src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/AnsiConsoleTheme.cs

```diff
@@ -47,10 +47,10 @@ public class AnsiConsoleTheme : ConsoleTheme
         /// Construct a theme given a set of styles.
         /// </summary>
         /// <param name="styles">Styles to apply within the theme.</param>
+        /// <exception cref="ArgumentNullException">When <paramref name="styles"/> is <code>null</code></exception>
         public AnsiConsoleTheme(IReadOnlyDictionary<ConsoleThemeStyle, string> styles)
         {
-            if (styles == null) throw new ArgumentNullException(nameof(styles));
-            _styles = styles.ToDictionary(kv => kv.Key, kv => kv.Value);
+            _styles = styles?.ToDictionary(kv => kv.Key, kv => kv.Value) ?? throw new ArgumentNullException(nameof(styles));
```

Comment:
The flow seems a bit less clear to me in this form. For example, we'd end up with an `ArgumentNullException()` if `ToDictionary()` (for some unknown reason) returned `null`. Could we leave out this change, and the similar change below?